kbase-extension/static/kbase/templates/data_staging/ftp_file_table.html

```diff
@@ -1,41 +1,46 @@
-<table id="kb-data-staging-table" class="table table-striped table-condensed" style="table-layout:fixed">
-    <thead>
+<table class="kb-staging-table table table-striped table-condensed">
+    <thead class="kb-staging-table-header">
         <tr>
-            <th style="width: 5%"></th>
-            <th style="width: 40%; overflow:hidden">Name</th>
-            <th style="width: 13%">Size</th>
-            <th style="width: 12%">Age</th>
-            <th style="width: 30%">Import As...</th>
+            <!-- TODO: what behavior should select all checkbox have? -->
+            <th class="kb-staging-table-header__checkbox">
+                <input type="checkbox" name="select_all" value="1" id="">
```

Comment:
is this the malformed id that the code checker is complaining about?

Reply:
yeah - i have plans to fix it once i get deeper into the implementation of the checkbox -> import selected button behavior